### PR TITLE
add faq question is_it_possible_to_tag_a_service_without_value

### PR DESCRIPTION
**Pull request by robinacar**

### CODE REVIEW

**Comment by tischrei** [comment]
recheck

File: umn/source/faqs/is_it_possible_to_tag_a_service_without_value [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:original_name: tms_faq_0009.html
+
+.. _tms_faq_0009:
+
+Is it possible to Tag a Service without value?
+============
```
Comment (tischrei) [inline umn/source/faqs/is_it_possible_to_tag_a_service_without_value:6]: Not enough "=" characters.

File: umn/source/faqs/is_it_possible_to_tag_a_service_without_value [hunk 2]
```diff
@@ -0,0 +1,9 @@
+:original_name: tms_faq_0009.html
```
Comment (tischrei) [inline umn/source/faqs/is_it_possible_to_tag_a_service_without_value:1]: The file misses the .rst postfix. Otherwise it will not be found.

**Review by tischrei — CHANGES_REQUESTED** [review]
Rename the file and add enough "=" character to the mentioned headline.

**Comment by vladimirhasko** [comment]
the problem is here:

2023-10-16 08:35:02.631285 | ubuntu-jammy | /home/zuul/src/github.com/opentelekomcloud-docs/tag-management-service/umn/source/faqs/index.rst:14: WARNING: undefined label: 'tms_faq_0009'
2023-10-16 08:35:01.736087 | ubuntu-jammy | /home/zuul/src/github.com/opentelekomcloud-docs/tag-management-service/umn/source/faqs/index.rst:16: WARNING: toctree contains reference to nonexisting document 'faqs/is_it_possible_to_tag_a_service_without_value'

**Comment by robinacar** [comment]
Hi Tino, I just updated the pull request. Please review as soon as possible.

**Review by tischrei — APPROVED** [review]

**Comment by tischrei** [comment]
@robinacar you can merge it

**Comment by mrumpler3** [comment]
Hi @robinacar ,

we will add the following FAQ to the documentation:

image

This FAQ is different than you suggested.

**The reason is:**
The behavior that you described is a bug, which should not be documented rather fix it. If you use the 'N/A' tag value by purpose and you also use empty tag value for the same tag key, then you will not be able to distinguish your resources in the Financial Dashboard based on tags. Empty tag values should not be converted to anything else, these should be kept as empty Strings.

Please open an incident for this issue, because we cannot add this faulty behavior to our service documentation. The issue is at the billing chain side where this conversion happens, TMS service cannot do anything to resolve this.

Therefore I close this proposed pull request.

**Pull request closed without merging**